### ISSUE TITLE
[EN][Proffesions][Core mechanic]Missing herb nodes in Nortrend

### ISSUE THREAD

**Issue by madKinno** [Atlantiss/BugTracker#3656]
Current situation with herbing nodes in Northrend is pretty bad, nodes are rare and often misplaced which causes prices of herbs to be pretty high. I compiled the list of all Northrend herbs and their Blizzlike positioning. Apologies if i forgot to add something 

Goldclover: should be found in Borean Tundra, Howling Fjord and Dragonblight which is not the case at the moment 
You can see how nodes should be placed here http://www.wowhead.com/object=189973/goldclover

Firethorn: should be found in Borean Tundra (near the Geyser field only). Correct herb nodes can be found here http://www.wowhead.com/object=191303/firethorn

Talandra’s Rose: Correct herb nodes can be found here http://www.wowhead.com/object=190170/talandras-rose

Tiger Lily: should be found in Borean Tundra, Grizzly Hills, Howling Fjord and Sholazar Basin. Correct herb nodes can be found here http://www.wowhead.com/object=190169/tiger-lily

Frozen Herb should be found in  Dragonblight, Zul'Drak , Wintergrasp, Hillsbrad Foothills and The Nexus . Correct herb nodes can be found here http://www.wowhead.com/object=190173/frozen-herb

Lichbloom should be be found in The Storm Peaks, Icecrown, Wintergrasp, Utgarde Pinnacle and Zul'Drak. Correct herb nodes can be found here  http://www.wowhead.com/object=190171/lichbloom

Adder’s Tongue should  be found in Sholazar Basin, Drak'Tharon Keep, Gundrak) and Ulduar. Correct herb nodes can be found here http://www.wowhead.com/object=191019/adders-tongue

Icethorn be found in The Storm Peaks, Icecrown , Wintergrasp , Utgarde Pinnacle, The Oculus and Zul'Drak. Correct herb nodes can be found here http://www.wowhead.com/object=190172/icethorn

I saw that separate reports on this matter are already reported but i decided to make this one since i think that it might be easier for devs to find everything in one spot.

**Comment by PARANOiA32** [comment]
This is still an issue!

**Comment by Simgiov** [comment]
Leveling alchemy is too hard because of WotLK levels, still waiting for fix.

**Comment by tomatosz** [comment]
I've been able to find Firethorn, no luck on the Talandra's Rose though.

**Comment by worloq** [comment]
i was gathering herbs today and this still is a big issue there is very few herbs in northrend and i found only tiger lilly some adders tongue and couple goldclover no talandra's rose witch is needed to northrend research in alchemy so this is big issue still i hope it will be corrected soon

**Comment by sigmundfreud2016** [comment]
True, Talandra's Rose can only be gathered from frozen herb. Not a single node in Zul'drak.
